content/integrations/jenkins.md

```diff
@@ -6,32 +6,36 @@ git_integration_title: jenkins
 newhlevel: true
 ---
 
+{{< img src="integrations/jenkins/integration-jenkins-overview.png" alt="Jenkins Plugin" >}}
+
 ## Overview
 
-{{< img src="integrations/jenkins/integration-jenkins-overview.png" alt="Jenkins Plugin" >}}
-A Jenkins plugin used to forward metrics, events, and service checks to an account at Datadog, automatically.
+This Jenkins plugin sends your build and deployment events to Datadog. From there, you can overlay them onto graphs of your other metrics so you can identify which deployments really affect your application's performance and reliability—for better or worse.
 
-***Note: The Jenkins Agent Integration is Deprecated***: The integration in the Datadog agent for Jenkins has been deprecated in favor of the open-source Jenkins plugin. This document refers to installing the Jenkins plugin. For more details about the plugin, refer to the [GitHub repo](https://github.com/DataDog/jenkins-datadog-plugin).
+The plugin also tracks build times (as a metric) and statuses (as a service check), so you'll know when your builds aren't healthy.
+
+**Note: The Jenkins check for the Datadog Agent is deprecated.** Use the [Jenkins plugin](https://github.com/DataDog/jenkins-datadog-plugin).
```

Comment:
Come to think of it, I don't think we should link to the code here. It tempts the reader to jump to the Github doc right before starting to read _this_ doc.

We _should_ link to it somewhere. Further Reading?